src/AddTodoForm.js

```diff
@@ -1,19 +1,29 @@
 import React from "react";
 
 const AddTodoForm = (props) => {
+    const {onAddTodo,} = props;
+    const [todoTitle, setTodoTitle] = React.useState();    
```

Comment:
I think you need to add an empty string inside your React.useState(); Otherwise it'll throw an error because it starts as undefined then gets updated.